docs/NationalInstruments/flot-highlights.md

```diff
@@ -0,0 +1,70 @@
+# flot-highlights-plugin
+
+This is a plugin for flot to highight points, bars and lines.
+
+Options
+-------
+
+The plugin supports these options:
+
+```javascript
+    highlights: [
+        {
+            selectedRange: [],
+            selectedIndexes: [],
+            show: false,
+            highlightLineWidth: 1,
+            highlightColor: '#ffffff',
+            highlightPoints: true,
+            highlightLines: false,
+            highlightBars: false,
+        },
+    ]
+```
+
+**selectedRange** the selected index range inside which points will be highlighted.
```

Comment:
Can you explain here about how to specify ranges and indexes per-series?

Reply:
OK